sql/core/src/main/scala/org/apache/spark/sql/execution/SparkPlan.scala

```diff
@@ -516,11 +543,28 @@ trait UnaryExecNode extends SparkPlan {
   def child: SparkPlan
 
   override final def children: Seq[SparkPlan] = child :: Nil
+  override def verboseString(
+      planToOperatorID : mutable.LinkedHashMap[TreeNode[_], Int],
+      codegenId: Option[Int]): String = {
+    s"""
+       |(${operatorIdStr(planToOperatorID)}) $nodeName ${wholestageCodegenIdStr(codegenId)}
+       |Input: ${child.output.mkString("[", ", ", "]")}
+     """.stripMargin
+  }
 }
 
 trait BinaryExecNode extends SparkPlan {
   def left: SparkPlan
   def right: SparkPlan
 
   override final def children: Seq[SparkPlan] = Seq(left, right)
+  override def verboseString(
+      planToOperatorID: mutable.LinkedHashMap[TreeNode[_], Int],
+      codegenId: Option[Int]): String = {
+    s"""
+       |(${operatorIdStr(planToOperatorID)}) $nodeName ${wholestageCodegenIdStr(codegenId)}
+       |Left output: ${left.output.mkString("[", ", ", "]")}
+       |Right output: ${right.output.mkString("[", ", ", "]")}
```

Comment:
@cloud-fan For binary exec, we again show the input which is basically left's output and right's output. ok ?